recipe/meta.yaml

```
@@ -137,6 +137,7 @@ outputs:
       detect_binary_files_with_prefix: False
       track_features:
         - sysroot_{{ cross_target_platform }}_{{ version }}
+        - sysroot_{{ cross_target_platform }}_{{ version }}_extra_weight
```

In case we need to do more of these in the future, should we adopt a pattern that lends itself to incrementing? Perhaps like this?

```suggestion
        - sysroot_{{ cross_target_platform }}_{{ version }}_feature_2
```